squire_lib/src/accounts.rs

```diff
@@ -0,0 +1,20 @@
+use uuid::Uuid;
+use settings;
+
+struct squire_account {
+    display_name: String,
+    user_name: String,
+    user_id: Uuid,
+    do_share: bool,
```

Comment:
We probably want to support a more fine-grain approach. Players should be able to specify that they are ok with everything being shared, just their decklist, or nothing. I would recommend making this a basic `enum` instead of a bool. There might be other things we want to think about, but that's a start.